test/inline_asm_constraints.cl

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown -x cl -cl-std=CL2.0 -O0 -fdeclare-opencl-builtins -finclude-default-header -emit-llvm-bc %s -o %t.bc
+// RUN: llvm-spirv %t.bc -spirv-ext=+SPV_INTEL_inline_assembly -spirv-text -o %t.spt
```

Comment:
Could you please apply the same pattern for this test?

- generate text form from binary form (`-spirv-ext` can be omitted at this stage)
- all extensions are enabled by default when reading SPIR-V, no need in `-spirv-ext` in command with `-r` flag

Reply:
Done. Also found and fixed subtle bug in llvm-spirv that led to difference when doing bc->spvtext and bc->spv->spvtext